src/models/ui.ts

```diff
@@ -59,6 +59,7 @@ export type LeftMenuSelectionType =
   | 'images-pane'
   | 'validation-pane'
   | 'search';
+export type LeftMenuBottomSelectionType = 'terminal' | null;
```

Comment:
I think I would've rather typed this as `'terminal' | undefined` since we haven't really used `null` in the project, but if it's useful here we can leave it like that

Reply:
This is needed so we can deselect the options from the bottom menu, after one of them was selected already